python/tests/test_json.py

```diff
@@ -64,8 +65,10 @@ def test_json(self):
             base_netlist_file,
         )
 
+        mock_file = MockFile()
+        mock_file.write(TEST_NETLIST)
         netlist = Netlist.create(
-            base_netlist_file=BaseNetlistFile.create(MockFile()),
+            base_netlist_file=BaseNetlistFile.create(mock_file),
```

Comment:
Needed to tweek this because I made netlist error if it tries to create one from an empty file